bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.module.ts

```diff
@@ -38,6 +39,7 @@ import { DtInputModule } from '@dynatrace/barista-components/input';
     KtbSliBreakdownModule,
     MatDialogModule,
     MomentModule,
+    DtCardModule,
```

Comment:
Is this module even used somewhere?

Reply:
No, actually not. I was trying out how it would look like as cards, but actually not using it in the end.